include/deal.II/numerics/vector_tools.h

```diff
@@ -857,6 +859,46 @@ namespace VectorTools
                 const bool                          project_to_boundary_first = false);
 
   /**
+   * The same as above  for distributed Triangulations.
+   * The underlying MatrixFree implementation requires to specify the number
+   * of components and the degree of the used FiniteElement as additional
+   * template parameters.
+   * Currently, the implementation only allows for @p enforce_zero_boundary
+   * and @p project_to_boundary_first to be <tt>false</tt>.
+   */
```

Comment:
please state that `vec_result` should be fully distributed, if I understand it correctly.